README.MD

```diff
@@ -1 +1,47 @@
 # GPostCard
+
+### Project Description
+
+GPostcard is a Web application that can generate postcards based on the text that is on them. After a postcard is generated, it can be sent to any Gmail user. By ticking the location, the user can get on the postcard a representative photo for their current location. Moreover, the user can record a message which will be automatically filled into the text area allowing accessability. 
+
+#### Goals
+
+* Text analysis
+* Image analysis
+* Generate relevant postcards
+* Design variability
+* Use API pre-trained models
+
+#### NON-Goals
+
+* Store user data
+* Use images that are not free for comercial use
```

Comment:
I am not so sure. Are non-goals something like useful features that we don't plan to implement (because we don't have time or the problem is too complex)? Or do non-goals refer to potential problems (e.g. security issues, using copyrighted images)?

Reply:
Okay, I see non-goals also like goals but the negation just emphasizes the importance of these. We can add these to to goals like "Not store data", "Use images free for commercial use". I think you can see them as potential problems that is our goal to avoid. Does it make sense?